QUICKSTART.md

```diff
@@ -13,13 +13,13 @@
 This process is the same if you launched the Dremio locally or via docker.
 
```

Comment:
Instructions need to be changed here to reflect passing params through conf file